### PR TITLE
py-pyvcf: new package

### PR DESCRIPTION
**Pull request by peetsv**
A VCFv4.0 and 4.1 parser for Python.

Prerequisite for MetaSV.

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-pyvcf/package.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+#
+# This is a template package file for Spack.  We've put "FIXME"
```
Comment (davydden) [inline var/spack/repos/builtin/packages/py-pyvcf/package.py]: please remove those left-overs

File: var/spack/repos/builtin/packages/py-pyvcf/package.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+# If you submit this package back to Spack as a pull request,
+# please first remove this boilerplate and all FIXME comments.
+#
+from spack import *
+
+
+class PyPyvcf(PythonPackage):
+    """PyVCF - A Variant Call Format Parser for Python"""
+
+    # FIXME: Add a proper url for your package's homepage here.
```
Comment (davydden) [inline var/spack/repos/builtin/packages/py-pyvcf/package.py]: remove that.

File: var/spack/repos/builtin/packages/py-pyvcf/package.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    homepage = "https://github.com/jamescasbon/PyVCF"
+    url      = "https://github.com/jamescasbon/PyVCF/archive/v0.6.0.tar.gz"
+
+    version('0.6.0', '5e38a610b1bf837e5e74a748e281cec7')
+
+    # depends_on('py-setuptools', type='build')
```
Comment (davydden) [inline var/spack/repos/builtin/packages/py-pyvcf/package.py]: if u don't need this, remove

**Comment by peetsv** [comment]
I will try again.

**Pull request closed without merging**